scripts/build_containers.sh

```diff
@@ -28,6 +28,7 @@ function create_pr_containers {
   docker build --target production \
                --cache-from="${CONTAINER_REPO}"/"${CONTAINER_APP}":builder-"${PULL_REQUEST}" \
                --cache-from="${CONTAINER_REPO}"/"${CONTAINER_APP}":pr-"${PULL_REQUEST}" \
+               --label quay.expires-after=8w \
```

Comment:
"when building a container for a PR - after 8 week quay may prune the tag" - makes sense (if we can't do anything more precise, that's plenty of wiggle room).

Reply:
This was a mistake. I didn't mean to add it to the production tag